tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/OpenTelemetry/StartSpanIntegration.cs

```diff
@@ -0,0 +1,79 @@
+// <copyright file="StartSpanIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.OpenTelemetry
+{
+    /// <summary>
+    /// Msmq calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyName = "OpenTelemetry.Api",
+        TypeName = "OpenTelemetry.Trace.Tracer",
+        MethodName = "StartSpan",
+        ReturnTypeName = "OpenTelemetry.Trace.TelemetrySpan",
+        ParameterTypeNames = new[] { ClrNames.String, "OpenTelemetry.Trace.SpanKind", "OpenTelemetry.Trace.SpanContext&", "OpenTelemetry.Trace.SpanAttributes", "System.Collections.Generic.IEnumerable`1[OpenTelemetry.Trace.Link]", "System.DateTimeOffset" },
+        MinimumVersion = "1.0.0",
+        MaximumVersion = "1.0.0",
```

Comment:
Same, why 1.0.0?

Reply:
This is the same reasoning as for StartRootSpanIntegration, the assembly version is at 1.0.0. I expect this will only change on the next major release so this should be fine

Reply:
Like `Newtonsoft.Json`, they are probably following the guidance from [Library guidance > Assembly versioning](https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/versioning#assembly-version):

> ✔️ CONSIDER only including a major version in the AssemblyVersion.
> ✔️ CONSIDER keeping the major version number of the AssemblyVersion and the NuGet package version in sync.